config/urls.py

```diff
@@ -113,3 +113,10 @@
         path("", include(wagtail_urls)),
     ]
 )
+
+if settings.DEBUG:
+    import debug_toolbar
+
+    urlpatterns = [
+        path("__debug__/", include(debug_toolbar.urls)),
+    ] + urlpatterns
```

Comment:
```suggestion
if apps.is_installed("debug_toolbar")::
    urlpatterns = [
        path("__debug__/", include("debug_toolbar.urls")),
    ] + urlpatterns
```

Reply:
@TNA-Allan This may require you to add `from django.apps import apps` to the top of the file also

Reply:
Nice one. Thanks Andy. Done